Externals/libusb/CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ add_library(usb SHARED EXCLUDE_FROM_ALL
     libusb/sync.c
 )
 set_target_properties(usb PROPERTIES VERSION 1.0.19)
+if(WIN32)
+    target_include_directories(usb BEFORE PUBLIC libusb PRIVATE msvc)
```

Comment:
It is to include a "config.h". That's too generic, so putting it BEFORE the other include directories is way safer!
We currently add all the include directories for all the external libraries in the global scope and it's propagated to all of them. Until we remove all the global options and move to direct dependencies, we'll need things like that.